### PR TITLE
ci-e2e-upgrade: Remove setting CLI vsn

### PR DESCRIPTION
**Pull request by brb**
It's controlled by set-env-variables.

### CODE REVIEW

**Comment by brb** [comment]
Blocked by https://github.com/cilium/cilium-cli/pull/2129.

**Review by brlbil — APPROVED** [review]

File: .github/workflows/tests-e2e-upgrade.yaml [hunk 1]
```diff
@@ -52,7 +52,6 @@ concurrency:
 
 env:
   # renovate: datasource=github-releases depName=cilium/cilium-cli
```
Comment (aanm) [inline .github/workflows/tests-e2e-upgrade.yaml]: I think we need to remove this line as well

**Review by aanm — CHANGES_REQUESTED** [review]

**Review by aanm — APPROVED** [review]

**Pull request merged**